core/src/main/resources/jenkins/console/ConsoleUrlProviderGlobalConfiguration/config.jelly

```diff
@@ -0,0 +1,13 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <j:if test="${descriptor.enabled}">
+        <f:section title="${%Console URL Provider}">
+            <f:block>
+                ${%description}
+            </f:block>
+            <f:entry>
+                <f:repeatableHeteroProperty field="providers" hasHeader="true" oneEach="true" header="${%Providers}" />
```

Comment:
It's a bit awkward that the default appears in the global list. I suggest this be filtered to not show Default explicitly (by implementing `#getProvidersDescriptors` AFAICT).

Reply:
Nice, thanks for the pointer to the expected pattern for the method name. Done in [64c3faa](https://github.com/jenkinsci/jenkins/pull/8321/commits/64c3faa8ac16742878377db5f9b0d14ff95a1338).